tools/test-proxy/CONTRIBUTING.md

```diff
@@ -0,0 +1,132 @@
+# Azure SDK Test-Proxy Contribution guide
+
+Within this folder are all the components that make up the the tool `Azure.Sdk.Tools.TestProxy`, henceforth referred to as the `test-proxy`.
+
+This product is currently developed on Visual Studio 2022, but CLI build/test also work fine. Currently, this project is built against `.net 6.0.403`.
```

Comment:
Will the version get out of date quickly, given we pin to the major version only in the csproj?

Reply:
It's pinned in our `globals.json`. The framework is only partially pinned in the csproj. I highlight how to fix that further on in the doc.

Reply:
I'm thinking we just support .NET 7 too. Skip the issues :P